airwallex/src/main/java/com/airwallex/android/view/util/PaymentMethodTypeInfoExtensions.kt

```diff
@@ -0,0 +1,39 @@
+package com.airwallex.android.view.util
+
+import com.airwallex.android.core.model.AirwallexPaymentRequestFlow
+import com.airwallex.android.core.model.DynamicSchemaField
+import com.airwallex.android.core.model.PaymentMethodTypeInfo
+import com.airwallex.android.core.model.TransactionMode
+import com.airwallex.android.view.PaymentMethodsViewModel.Companion.FLOW
+
+fun PaymentMethodTypeInfo.filterRequiredFields(): List<DynamicSchemaField>? {
+    return this
+        .fieldSchemas
+        ?.firstOrNull { schema -> schema.transactionMode == TransactionMode.ONE_OFF }
+        ?.fields
+        ?.filter { !it.hidden }
+}
+
+fun PaymentMethodTypeInfo.fetchPaymentFlow(): AirwallexPaymentRequestFlow {
```

Comment:
I feel `fetch` is more like a request, since this is just data transformation, we may just call `toPaymentFlow`.

Reply:
ok